templates/page.tpl.php

```diff
@@ -87,18 +87,15 @@
 </main>
 
 <footer role="contentinfo">
-  <section class="page-footer">
-    <div class="wrapper">
-      <section class="footer-top">
-        <nav>
-          <?php print render($page['footer']); ?>
-        </nav>
-      </section>
-    </div>
-  </section>
-  <section class="page-bottom">
-    <div class="wrapper">
+  <div class="wrapper">
+    <section class="footer-top">
+      <?php print render($page['footer_top']); ?>
+    </section>
+    <section class="footer-bottom">
+      <?php print render($page['footer_bottom']); ?>
```

Comment:
It is exactly how we implemented the inline ul in the footer. Having the class in the region will make all the ul in the footer bottom inline by default. To change it, people will need to use the same way to remove the class. 

I'd prefer to leave it without the class in base theme and implement the inline style in subtheme.
In this case, I'm happy with the PR.